src/core/client/admin/routes/Moderate/ModerateNavigation/Navigation.tsx

```diff
@@ -20,6 +20,7 @@ interface Props {
   section?: SectionFilter | null;
   router: Router;
   match: Match;
+  mode?: "PRE" | "POST" | "%future added value" | null;
```

Comment:
Not a comment to change anything here, but we probably need a rule/pattern to handle these in the future.

Any suggestions @cvle?

Reply:
What do you mean exactly?

Reply:
It's common for us to re-use enum's for components, and having to copy the `"%future added value" | null` everywhere feels complex. Do we have a more universal way to generate enum types that include this `"%future added value"` already?